game.js

```diff
@@ -72,7 +60,7 @@ Game.prototype.checkWinner = function(combo){
 
 
 //shuffle order
-var players = [Player1,Player2];
+var players = [new HumanPlayer(), new ComputerPlayer()];
```

Comment:
This, in addition the the problem with the function Game() constructor are the two main problems you are running into. The rest is suggestions and ideas.